specs.rst

```diff
@@ -7,10 +7,10 @@ Rationale and specifications of the libblockdev library
 Scope
 ------
 
-libblockdev library [1]_ will be a new library for doing operations with block
-devices replacing and extending the code from the blivet.devicelibs
-(python-blivet) and ssmlib.backends (storagemanager) modules. It's expected to
-support the following storage technologies:
+libblockdev library is a library for doing operations with block devices
+replacing and extending the code from the blivet.devicelibs (python-blivet) and
+ssmlib.backends (storagemanager) modules. It supports the following storage
```

Comment:
Unfortunately, ssm hasn't been ported to libblockdev so far. (also applies to the last change below)